README.md

```diff
@@ -1,21 +1,107 @@
-# Website
+# Get Started
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-## Installation
+Docusaurus is a **static-site generator**. It builds a **single-page application** with fast client-side navigation, leveraging the full power of **React** to make your site interactive. It provides out-of-the-box **documentation features** but can be used to create **any kind of site** (personal website, product, blog, marketing landing pages, etc).
 
-```console
-yarn install
+## Dependencies
+
+* **Nodejs** version >= 14 or above
+
+* **Yarn** version >= 1.5
+
+* **NPM** [Guide](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+* **git**
+
+
+## How to install
+
+## Install Dependencies for Windows users
+
+Download [Nodejs installer](https://nodejs.org/en/download/) from their Official website.
+
+### Install yarn
+
+```
+npm install --global yarn
+```
+
+### Install Git
+
+Download [Git Installer](https://git-scm.com/download/win) from their Official website.
+
+## Install Dependencies for Mac Users
+
+Download [Nodejs installer](https://nodejs.org/en/download/) from their Official website.
+
+Alternatively:
+
+### Using Homebew
+
+```
+brew install node
+```
+
+### Using MacPorts
+
+```
+sudo port install nodejs17
+```
+
+### Install yarn
+
+```
+npm install --global yarn
+```
+
+### Install Git
+
+There are several ways to install Git on a Mac. The easiest is probably to install the Xcode Command Line Tools. Latest macOS has git preinstalled.
```

Comment:
```suggestion
Latest macOS has git preinstalled. Otherwise, there are several ways to install Git on a Mac. The easiest is to install the Xcode Command Line Tools.
```
I also suggest replacing "latest macOS" with the actual version it started being pre-installed.

Reply:
I removed the Latest macOS.